openapi.yaml

```diff
@@ -292,46 +284,68 @@ paths:
               schema:
                 $ref: "#/components/schemas/PassportResponse"
         "400":
-          description: "Invalid name supplied"
+          description: "Invalid data"
         "404":
           description: "Passport not found"
 
-  /lostPassport:
-    post:
+  /person/{personId}/passport/{id}/lostPassport:
+    patch:
       tags:
         - "LostPassport"
-      summary: "change status of current passport and create a new active"
-      description: "Find passport Person`s data (deactivate it) and create a new one for this Person"
+      summary: "change status of current passport"
+      description: "Deactivate person`s passport "
       operationId: "changeStatusOfPassport"
-      requestBody:
-        content:
-          'application/json':
-            schema:
-              $ref: "#/components/schemas/PersonRequest"
+      parameters:
+        - name: "personId"
+          in: "path"
+          description: "Person`s ID to get"
+          required: true
+          schema:
+            type: "string"
+        - name: "id"
+          in: "path"
+          description: "Person`s ID to get"
+          required: true
+          schema:
+            type: "string"
+        - name: "active"
+          in: "query"
+          description: "Passport deactivation status"
+          required: true
+          schema:
+            type: "boolean"
+            enum:
+              - false
       responses:
         "200":
-          description: "Successful operation: new passport created"
+          description: "Successful operation: you need to create a new passport"
           content:
             '*/*':
               schema:
-                items:
-                  $ref: "#/components/schemas/PassportResponse"
-        "404":
-          description: "Person not found"
+                $ref: "#/components/schemas/PassportResponse"
+        "400":
+          description: "Invalid data"
 
-  /searches:
+  /person/{id}/searches:
     post:
```

Comment:
here changed endpoint to make access for this search only for auth user) Here we can add more function if we add some roles...

Reply:
It's not clear what is `id` here and where to get if from if we have only a passport number.

Reply:
fixed endpoint returned